ci/infra/libvirt/terraform.tfvars.json.ci.example

```diff
@@ -1,14 +1,13 @@
 {
-    "libvirt_uri": "qemu:///system",
+    "libvirt_uri": "qemu+ssh://jenkins@node52.cloud.suse.de/system",
     "pool": "default",
-    "image_uri": "SLES15-SP1-JeOS.x86_64-15.1-OpenStack-Cloud-QU1.qcow2",
+    "image_uri": "http://download.suse.de/install/SLE-15-SP1-JeOS-QU2/SLES15-SP1-JeOS.x86_64-15.1-OpenStack-Cloud-QU2.qcow2",
     "stack_name": "testing",
-    "network_cidr": "10.17.0.0/22",
```

Comment:
why are you changing this? this must match the range defined in the ip route to the kwm host

Reply:
I know, that's why I used the values that appeared in the docs for the jump host in https://github.com/SUSE/caasp-automation/tree/master/ecp-nbg-jumphost

Reply:
I think that is the network for the VMware VMs. The kvm host is used to provide the route. We should update that document to reflect that now there are two VM networks: one for VMware and other for libivirt.